[2022-02-03 19:02:08,756] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-02-02T09:00:00+00:00 [queued]>
[2022-02-03 19:02:08,767] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-02-02T09:00:00+00:00 [queued]>
[2022-02-03 19:02:08,767] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 19:02:08,767] {taskinstance.py:881} INFO - Starting attempt 1 of 1
[2022-02-03 19:02:08,767] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 19:02:08,774] {taskinstance.py:901} INFO - Executing <Task(TwitterOperator): twitter_aluraonline> on 2022-02-02T09:00:00+00:00
[2022-02-03 19:02:08,787] {standard_task_runner.py:54} INFO - Started process 10739 to run task
[2022-02-03 19:02:08,860] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'twitter_dag', 'twitter_aluraonline', '2022-02-02T09:00:00+00:00', '--job_id', '17', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/twitter_dag.py', '--cfg_path', '/tmp/tmp4xsb5vmj']
[2022-02-03 19:02:08,861] {standard_task_runner.py:78} INFO - Job 17: Subtask twitter_aluraonline
[2022-02-03 19:02:08,899] {logging_mixin.py:112} INFO - Running <TaskInstance: twitter_dag.twitter_aluraonline 2022-02-02T09:00:00+00:00 [running]> on host SPVALN1-612R9D4.pitagoras.apollo.br
[2022-02-03 19:02:08,960] {base_hook.py:89} INFO - Using connection to: id: twitter_default. Host: https://api.twitter.com, Port: None, Schema: None, Login: None, Password: None, extra: XXXXXXXX
[2022-02-03 19:02:08,962] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-02-02T09:00:00.00Z&end_time=2022-02-03T09:00:00.00Z
[2022-02-03 19:02:12,514] {logging_mixin.py:112} WARNING - /mnt/d/Alura/engenharia-dados-apache-airflow/datapipeline/.env/lib/python3.7/site-packages/urllib3/connectionpool.py:988: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
[2022-02-03 19:02:12,783] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-02-02T09:00:00.00Z&end_time=2022-02-03T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe48hq4pr9ybapa6dlmyhdlmw67b1
[2022-02-03 19:02:12,784] {logging_mixin.py:112} WARNING - /mnt/d/Alura/engenharia-dados-apache-airflow/datapipeline/.env/lib/python3.7/site-packages/urllib3/connectionpool.py:988: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
[2022-02-03 19:02:13,088] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-02-02T09:00:00.00Z&end_time=2022-02-03T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe48hq2mcvox139a9f0wt5ze6jou5
[2022-02-03 19:02:13,089] {logging_mixin.py:112} WARNING - /mnt/d/Alura/engenharia-dados-apache-airflow/datapipeline/.env/lib/python3.7/site-packages/urllib3/connectionpool.py:988: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
[2022-02-03 19:02:13,351] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-02-02T09:00:00.00Z&end_time=2022-02-03T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe48hnzozme40lsqecbnzb9655h19
[2022-02-03 19:02:13,351] {logging_mixin.py:112} WARNING - /mnt/d/Alura/engenharia-dados-apache-airflow/datapipeline/.env/lib/python3.7/site-packages/urllib3/connectionpool.py:988: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
[2022-02-03 19:02:13,624] {taskinstance.py:1070} INFO - Marking task as SUCCESS.dag_id=twitter_dag, task_id=twitter_aluraonline, execution_date=20220202T090000, start_date=20220203T220208, end_date=20220203T220213
[2022-02-03 19:02:13,742] {local_task_job.py:102} INFO - Task exited with return code 0
